readme.md

```diff
@@ -28,7 +28,7 @@ In addition to the basic functionality create a BEM modifier called `speed-up` f
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://liliazh.github.io/layout_stop-watch/)
```

Comment:
This link must be in the PR description
<img width="971" alt="image" src="https://github.com/mate-academy/layout_stop-watch/assets/64750958/6fe12edd-0893-454d-b281-bd46f7c23ca3">